src/components/ToolMenu/Draw/DeleteAllButton/index.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import {
+  useEffect, useState
+} from 'react';
+
+import { ExclamationCircleOutlined } from '@ant-design/icons';
+import { Modal } from 'antd';
+import OlGeometry from 'ol/geom/Geometry';
+import OlVectorLayer from 'ol/layer/Vector';
+import OlVectorSource from 'ol/source/Vector';
+
+import {
+  useTranslation
+} from 'react-i18next';
+
+import {SimpleButton, useMap} from '@terrestris/react-geo';
+import { SimpleButtonProps } from '@terrestris/react-geo/dist/Button/SimpleButton/SimpleButton';
+import { DigitizeUtil } from '@terrestris/react-util/dist/Util/DigitizeUtil';
+
+const { confirm } = Modal;
+
+interface OwnProps {
+  /**
+   * The vector layer which will be used for digitize features.
+   * The standard digitizeLayer can be retrieved via `DigitizeUtil.getDigitizeLayer(map)`.
+   */
+  digitizeLayer?: OlVectorLayer<OlVectorSource<OlGeometry>>;
+}
+
+export type DeleteAllButtonProps = OwnProps & SimpleButtonProps;
+
+/**
+ * The className added to this component.
+ */
+const defaultClassName = 'deleteallbutton';
+
+export const DeleteAllButton: React.FC<DeleteAllButtonProps> = ({
+  className,
+  digitizeLayer,
+  ...passThroughProps
+}) => {
+  const [layers, setLayers] = useState<[OlVectorLayer<OlVectorSource<OlGeometry>>]|null>(null);
+
+  const map = useMap();
+
+  const {
+    t
+  } = useTranslation();
+
+  useEffect(() => {
+    if (!map) {
+      return;
+    }
+
+    if (digitizeLayer) {
+      setLayers([digitizeLayer]);
+    } else {
+      setLayers([DigitizeUtil.getDigitizeLayer(map)]);
+    }
+  }, [map, digitizeLayer]);
+
+  if (!layers) {
+    return null;
+  }
+
+  const onClick = () => {
+
+    confirm({
+      title: (t('Draw.deleteAll')),
+      content: (t('Draw.confirmMessage')),
```

Comment:
The locale should be placed inside the corresponding locale object: `Draw` -> `DeleteAllButton`.

Reply:
done in: https://github.com/terrestris/shogun-gis-client/pull/1476/commits/28dbc8e5d678aa565d975ccdf1943ffa165c3b05